web/client/src/components/Scopes.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { MDBContainer, MDBCol, MDBBtn, MDBInput } from "mdbreact";
+
+/**
+ *
+ * Class that represents the page that contains the ability to choose the scopes
+ */
+class Scopes extends React.Component {
```

Comment:
I think we're using functional react for this project

Reply:
+1, in order to add state in functional components look into react hooks

Reply:
Done